composer.json

```diff
@@ -27,16 +27,16 @@
     "php": ">=5.4",
     "PrestaShop/smarty": "dev-master",
     "nikic/php-parser": "^2.1",
-    "twig/twig": "^1.34",
-    "symfony/twig-bridge": "^2.8"
+    "twig/twig": "^1.34|^2.0",
```

Comment:
The recommended official notation is `^1.34 || ^2.0`

Reply:
you're right, thanks